pkgs/tools/admin/lexicon/default.nix

```diff
@@ -1,44 +1,38 @@
 { lib
-, python3
+, buildPythonApplication
 , fetchFromGitHub
+, poetry-core
+, mock
+, pytestCheckHook
+, pytest-xdist
+, vcrpy
+, beautifulsoup4
+, boto3
+, cryptography
+, dnspython
+, future
+, localzone
+, oci
+, pynamecheap
+, pyyaml
+, requests
+, softlayer
+, tldextract
+, transip
+, xmltodict
+, zeep
 }:
 
-let
-  py = python3.override {
-    packageOverrides = self: super: {
-      # until https://github.com/ags-slc/localzone/issues/1 gets resolved
-      dnspython = super.dnspython.overridePythonAttrs(oldAttrs: rec {
-        pname = "dnspython";
-        version = "1.16.0";
-        # since name is defined from the previous derivation, need to override
-        # name explicity for correct version to show in drvName
-        name = "${pname}-${version}";
-
-        src = super.fetchPypi {
-          inherit pname version;
-          extension = "zip";
-          sha256 = "00cfamn97w2vhq3id87f10mjna8ag5yz5dw0cy5s0sa3ipiyii9n";
-        };
-      });
-
-      localzone = super.localzone.overridePythonAttrs(oldAttrs: rec {
-        meta = oldAttrs.meta // { broken = false; };
-      });
-    };
-  };
-in
-  with py.pkgs;
-
 buildPythonApplication rec {
```

Comment:
```suggestion
python3.pkgs.buildPythonApplication rec {
```

Reply:
Is there no issues with using with python3.pkgs instead?